katello/katello-backup

```diff
@@ -7,6 +7,8 @@ require 'date'
 @options = {}
 @dir = nil
 
+EXCLUDED="--exclude goferd,foreman-proxy,squid,qdrouterd,qpidd"
```

Comment:
Verified that smart_proxy_dynflow_core can also be excluded. Could you add it @lzap ? Thanks!